go.mod

```diff
@@ -2,6 +2,21 @@ module knative.dev/func
 
 go 1.20
 
+replace (
+	// Version used in Eventing
+	github.com/cloudevents/sdk-go/sql/v2 => github.com/cloudevents/sdk-go/sql/v2 v2.0.0-20220930150014-52b12276cc4a
```

Comment:
IMO that's probably covered on the Eventing side and the versions bubbled correctly. Then the replace shouldn't be needed anymore.